tfe/resource_tfe_workspace.go

```diff
@@ -482,6 +483,7 @@ func resourceTFEWorkspaceUpdate(d *schema.ResourceData, meta interface{}) error
 		if !ok {
 			_, err := tfeClient.Workspaces.RemoveVCSConnectionByID(ctx, id)
 			if err != nil {
+				d.Partial(true)
```

Comment:
I'm wondering if we should also use this below for error cases within changes to ssh_key_id, tag_names, remote_state_consumer_ids ?

Reply:
@sebasslash is there an answer to this question? 🤔